SA3/qa_web/templates/qa_web/answerspage.html

```diff
@@ -45,17 +45,42 @@
       <h1>{{ currentQuestion.title }}</h1>
       <p>{{ currentQuestion.content }}</p>
       <h2>Question by {{ currentQuestion.owner }} on {{ currentQuestion.creation_date }}</h2>
+      <hr>
   </div>
   <div id="answers">
   <!-- Display the answers -->
-    {% for answer in answers %}
-        <div id="answer_{{ answer.id }}">
-            <p>{{ answer.content }}</p>
-            <h3>Answer by {{ answer.owner }} on {{ answer.creation_date }}</h3>
-        </div>
-    {% empty %}
-      <h3>Be the first to answer this question!</h3>
-    {% endfor %}
+  {% for answer in bestAnswer %}
+      <form id="deselect{{ answer.id }}_form" method="post">
+          {% csrf_token %}
+          {% if user == currentQuestion.owner %}
+              <h3 class="good_answer">You have selected this as the best answer!</h3>
+          {% else %}
+              <h3 class="good_answer">{{currentQuestion.owner}} has selected this as the best answer!</h3>
+          {% endif %}
+          <p>{{ answer.content }}</p>
+          <!--Allow one to deselect the best answer if user is logged in and is the user who asked the question-->
+          {% if user.is_authenticated and user == currentQuestion.owner %}
+              <input type = "submit" name="deselect", value="Deselect as Best Answer"></button>
```

Comment:
Why is the `deselect` unlike the `select_{{ answer.id }}` in the sense that we do not need to specify the answer id? Just curious because I know it does not cause problems for the actual functionality.